main.go

```diff
@@ -65,7 +65,7 @@ func main() {
 		return
 	}
 
-	dbRepo, err := mongodb.NewRepository(cfg)
+	dbRepo, err := postgreSQL.NewPostgresDB(context.Background(), cfg.Database)
```

Comment:
When you have renamed the function and the package, you can it call here `postgres.NewRepository(...)`